pontoon/base/static/js/translate.js

```diff
@@ -198,6 +198,21 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Update current translation length and cache
+     *
+     * lengthOnly Do not update cache
+     */
+    updateCurrentLengthAndCache: function (lengthOnly) {
```

Comment:
Might wanna include the word "translation" in the name so we know what cache and what length we're updating.